CMakeLists.txt

```diff
@@ -52,44 +51,41 @@ add_subdirectory(${PROJECT_SOURCE_DIR}/Thirdparty/DBoW2/)
 add_subdirectory(${PROJECT_SOURCE_DIR}/Thirdparty/g2o/)
 
 add_library(ORB_SLAM2 SHARED
-src/System.cc
-src/Tracking.cc
-src/LocalMapping.cc
-src/LoopClosing.cc
-src/ORBextractor.cc
-src/ORBmatcher.cc
-src/FrameDrawer.cc
-src/Converter.cc
-src/MapPoint.cc
-src/KeyFrame.cc
-src/Map.cc
-src/MapDrawer.cc
-src/Optimizer.cc
-src/PnPsolver.cc
-src/Frame.cc
-src/KeyFrameDatabase.cc
-src/Sim3Solver.cc
-src/Initializer.cc
-src/Viewer.cc
+    src/System.cc
+    src/Tracking.cc
+    src/LocalMapping.cc
+    src/LoopClosing.cc
+    src/ORBextractor.cc
+    src/ORBmatcher.cc
+    src/Converter.cc
+    src/MapPoint.cc
+    src/KeyFrame.cc
+    src/Map.cc
+    src/Optimizer.cc
+    src/PnPsolver.cc
+    src/Frame.cc
+    src/KeyFrameDatabase.cc
+    src/Sim3Solver.cc
+    src/Initializer.cc
 )
 
 target_include_directories(ORB_SLAM2 PUBLIC
-${PROJECT_SOURCE_DIR}
-${PROJECT_SOURCE_DIR}/Thirdparty/DBoW2
-${PROJECT_SOURCE_DIR}/Thirdparty/g2o
-${PROJECT_SOURCE_DIR}/include
-${OpenCV_INCLUDE_DIRS}
-${EIGEN3_INCLUDE_DIR}
-${Pangolin_INCLUDE_DIR}
-${PROJECT_BINARY_DIR}
+    ${PROJECT_SOURCE_DIR}
+    ${PROJECT_SOURCE_DIR}/Thirdparty/DBoW2
+    ${PROJECT_SOURCE_DIR}/Thirdparty/g2o
+    ${PROJECT_SOURCE_DIR}/include
+    ${OpenCV_INCLUDE_DIRS}
+    ${EIGEN3_INCLUDE_DIR}
+    ${Pangolin_INCLUDE_DIR}
+    ${PROJECT_BINARY_DIR}
 )
 
 target_link_libraries(ORB_SLAM2
-${OpenCV_LIBS}
-${EIGEN3_LIBS}
-${Pangolin_LIBRARIES}
-DBoW2
-g2o
+    ${OpenCV_LIBS}
+    ${EIGEN3_LIBS}
+    ${Pangolin_LIBRARIES}
```

Comment:
Same here you have removed Pangolin libraries.

Reply:
oups, fixed in 14a723d783aa68df657814a0c75d9858909f8d06